src/main/java/lotto/controller/LottoGameController.java

```diff
@@ -8,36 +8,43 @@
 
 public class LottoGameController {
 
-    private final LottoGame lottoGame = new LottoGame();
-
     public void run() {
-        Lottos lottos = buyAutoLotto();
+        LottoAmount lottoAmount = makeLottoAmount();
 
-        Lotto winningLotto = askWinningLottoNumbers();
-        LottoNumber bonusNumber = InputView.askBonusNumber();
+        LottoGame lottoGame = buyTwoTypeLottos(lottoAmount);
+        printEachLottos(lottoGame);
 
-        WinningLotto lastWinningLotto = makeWinningLotto(winningLotto, bonusNumber);
+        WinningLotto lastWinningLotto = askWinningLotto();
+        LottoGameResult lottoGameResult = getRankOfLottoGame(lottoGame, lastWinningLotto);
```

Comment:
```suggestion
        LottoGameResult lottoGameResult = lottoGame.draw(lastWinningLotto);
```

와 같은 구조로 바꿔보면 어떨까요!?

Reply:
바꿔봤습니다. 컨트롤러에서 로직을 갖고 있는 것 보다 LottoGame이 갖고 있는게 확실히 더 맞는 방식이라 생각해요!